package.json

```diff
@@ -131,6 +131,7 @@
     "d3-shape": "^1.2.2",
     "d3-time-format": "^2.1.3",
     "dompurify": "1.0.9",
+    "emoji-flags": "^1.2.0",
```

Comment:
Shouldn't we remove `"react-world-flags": "1.2.4",` too?

Reply:
whoops, thanks